ratpack-cassandra/src/main/java/smartthings/ratpack/cassandra/FixedRetryPolicy.java

```diff
@@ -0,0 +1,67 @@
+package smartthings.ratpack.cassandra;
+
+import com.datastax.driver.core.Cluster;
+import com.datastax.driver.core.ConsistencyLevel;
+import com.datastax.driver.core.Statement;
+import com.datastax.driver.core.WriteType;
+import com.datastax.driver.core.exceptions.DriverException;
+import com.datastax.driver.core.policies.RetryPolicy;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * A policy that defines a default behavior to adopt when a request fails.
+ * Difference of existing DefaultRetryPolicy is that use Client side retry count variable only. and available retry count fixed 3 times.
+ */
+public class FixedRetryPolicy implements RetryPolicy {
+	private int retryCount = 3;
+
+	private Logger logger = LoggerFactory.getLogger(FixedRetryPolicy.class);
+
+	@Override
+	public RetryDecision onReadTimeout(Statement stmnt, ConsistencyLevel cl, int requiredResponses, int receivedResponses, boolean dataReceived, int nbRetry) {
+		logger.warn("[CustomRetryPolicy]-onReadTimeout(), requiredResponses={}, receivedResponses={}, dataReceived={}, nbRetry={}", requiredResponses, receivedResponses, dataReceived, nbRetry);
```

Comment:
Lets leave logging out and allow for composition with the logging retry policy

Reply:
What is mean? Can you explain more?